postcss.config.js

```diff
@@ -1,3 +1,3 @@
-const config = require('./packages/terra-clinical-site/postcss.config');
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
The default config handle this, so this can be removed

Reply:
the whole file?

Reply:
Yes the postcss configuration is defined in terra-dev-site's webpack config